src/promptflow/README.md

```diff
@@ -1,28 +1,112 @@
-> participating [discussions](https://github.com/microsoft/promptflow/discussions),
-> opening [issues](https://github.com/microsoft/promptflow/issues/new/choose),
-> submitting [PRs](https://github.com/microsoft/promptflow/pulls).
+> Welcome to join us to make Prompt flow!
 
 **Prompt flow** is a suite of development tools designed to streamline the end-to-end development cycle of LLM-based AI applications, from ideation, prototyping, testing, evaluation to production deployment and monitoring. It makes prompt engineering much easier and enables you to build LLM apps with production quality.
 
 With prompt flow, you will be able to:
 
-- Create executable workflows that link LLMs, prompts, Python code and other tools together.
-- Debug and iterate your flows, especially the interaction with LLMs with ease.
-- Evaluate your flow's quality and performance with larger datasets.
-- Integrate the testing and evaluation into your CI/CD system to ensure quality of your flow.
-- Deploy your flow to the serving platform you choose or integrate into your app's code base easily.
-- (Optional but highly recommended) Collaborate with your team by leveraging the cloud version of [Prompt flow in Azure AI](https://learn.microsoft.com/en-us/azure/machine-learning/prompt-flow/overview-what-is-prompt-flow?view=azureml-api-2).
+- **Create and Iteratively Develop Flow**
+    - Create executable workflows that link LLMs, prompts, Python code and other tools together.
+    - Debug and iterate your flows, especially the interaction with LLMs with ease.
+- **Evaluate Flow Quality and Performance**
+    - Evaluate your flow's quality and performance with larger datasets.
+    - Integrate the testing and evaluation into your CI/CD system to ensure quality of your flow.
+- **Streamlined Development Cycle for Production**
+    - Deploy your flow to the serving platform you choose or integrate into your app's code base easily.
+    - (Optional but highly recommended) Collaborate with your team by leveraging the cloud version of [Prompt flow in Azure AI](https://learn.microsoft.com/en-us/azure/machine-learning/prompt-flow/overview-what-is-prompt-flow?view=azureml-api-2).
+
+## Installation
+
+Ensure you have a python environment, `python=3.9` is recommended.
+
+```sh
+pip install promptflow promptflow-tools
+```
+
+## Quick Start ⚡
+
+<details>
+<summary><b>Create a chatbot with prompt flow</b></summary>
+ It creates a new <b>flow folder</b> named my_chatbot and generates the necessary flow files within it. The --flow argument is the path to the flow folder.
+</details>
+
+Initiate a prompt flow from a chat template:
+
+```sh
+pf flow init --flow ./my_chatbot --type chat
+```
+
+<details>
+<summary><b>Setup a connection for your API key</b></summary>
+Navigate to the `my_chatbot` folder, you can find a yaml file named `openai.yaml` file, which is the definition of the connection to store your Open AI key.
+</details>
+
+> <details>
```

Comment:
This readme is for pypi page, we can remove this to make content concise.

Reply:
done